QtOffscreenViews/QAndroidOffscreenWebView.h

```diff
@@ -91,6 +91,7 @@ class QAndroidOffscreenWebView
 
 	bool getIgnoreSslErrors() const { return ignore_ssl_errors_; }
 	void setIgnoreSslErrors(bool ignore) { ignore_ssl_errors_ = ignore; }
+    void setWebContentsDebuggingEnabled(bool enabled);
```

Comment:
Please fix the whitespace (tabs vs spaces?)

Reply:
Done.